app/views/content/is-teaching-right-for-me/get-school-experience.md

```diff
@@ -60,7 +61,7 @@ $get-school-experience$
 
 ### Teaching internships
 
-A paid [teaching internship](/teaching-internship-providers) could help you to understand what it’s really like in the classroom if you’re currently doing an undergraduate degree and are interested in teaching:
+A [paid teaching internship](/is-teaching-right-for-me/teaching-internship-providers) could help you to understand what it’s really like in the classroom if you’re currently doing an undergraduate degree and are interested in teaching:
```

Comment:
Wonder if it would be better to replace this inline link with a separate link at the end of this section along the lines of `Find out more about paid teaching internships`, to make sure they read the eligibility criteria first?

(Appreciate this is in prod already!)